lib/manifest.ps1

```diff
@@ -22,13 +22,47 @@ function url_manifest($url) {
     $str | convertfrom-json
 }
 
+function Get-Manifest($app) {
+    $bucket, $manifest, $url = $null
+    # check if app is a URL or UNC path
+    if ($app -match '^(ht|f)tps?://|\\\\') {
+        $url = $app
+        $app = appname_from_url $url
+        $manifest = url_manifest $url
+    } else {
+        $app, $bucket, $version = parse_app $app
+        if ($bucket) {
+            $manifest = manifest $app $bucket
+        } else {
+            foreach ($bucket in Get-LocalBucket) {
+                $manifest = manifest $app $bucket
+                if ($manifest) {
+                    break
+                }
+            }
+        }
+        if (!$manifest) {
+            # couldn't find app in buckets: check if it's a local path
+            if (!$app.EndsWith('.json')) {
+                $app += '.json'
```

Comment:
Was the [`$path = $app`](https://github.com/ScoopInstaller/Scoop/pull/4960/files#diff-6493ba74d7ec238c6750a2afe3965c6f33ac117cfbca02db2ccec2bf533b3168L95-L96) intentionally removed here?

```diff
-            $path = $app
-            if (!$path.endswith('.json')) { $path += '.json' }
-            if (Test-Path $path) {
-                $url = "$(Resolve-Path $path)"
-                $app = appname_from_url $url
-                $manifest, $bucket = url_manifest $url
-            }
+           if (!$app.EndsWith('.json')) {
+                 $app += '.json'
+           }
+           if (Test-Path $app) {
+                 $url = Convert-Path $app
+                 $app = appname_from_url $url
+                 $manifest = url_manifest $url
+           }
```

This causes a change printing `$app` with a `.json` suffix when the app does not exist.
before:
```
❯ scoop info extras/go-ipfs
Could not find manifest for 'extras/go-ipfs'.
```

after:
```
❯ scoop info extras/go-ipfs
Could not find manifest for 'go-ipfs.json' in local buckets.
```

Reply:
Fixed in #4981